src/common/transformations/src/transformations/common_optimizations/mul_conv_fusion.cpp

```diff
@@ -41,11 +71,12 @@ ov::pass::MultiplyConvolutionFusion::MultiplyConvolutionFusion() {
 
         const auto& weights_shape = weights.get_shape();
         const auto& mul_const_shape = mul_const.get_shape();
-        // Check if mul_const if broadcastable to weights.
+
+        // Check if mul_const is broadcastable to weights.
         // Also if mul_const's rank matches weights rank and mul_const.shape[0] != 1
         // then we can't fuse the multiply, since first dimension in mul_const corresponds to
         // batch size, while first dimension in weights corresponds to output channel count
-        if (op::util::check_for_broadcast(weights_shape, mul_const_shape) ||
+        if (!are_shapes_compatible(weights_shape, mul_const_shape) ||
```

Comment:
`op::util::check_for_broadcast` is also used in other places
please replace all usages with a new correct version.

Reply:
fixed.